[2023-12-27 18:12:09,653] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: OpenWeather_ETL2.envio_mail manual__2023-12-27T18:11:45.832652+00:00 [queued]>
[2023-12-27 18:12:09,680] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: OpenWeather_ETL2.envio_mail manual__2023-12-27T18:11:45.832652+00:00 [queued]>
[2023-12-27 18:12:09,681] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-27 18:12:09,682] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-27 18:12:09,683] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-27 18:12:09,718] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): envio_mail> on 2023-12-27 18:11:45.832652+00:00
[2023-12-27 18:12:09,729] {standard_task_runner.py:52} INFO - Started process 14195 to run task
[2023-12-27 18:12:09,738] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'OpenWeather_ETL2', 'envio_mail', 'manual__2023-12-27T18:11:45.832652+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/ETL_OpenWeather2.py', '--cfg-path', '/tmp/tmpb5bstxtb', '--error-file', '/tmp/tmp8m24ex8i']
[2023-12-27 18:12:09,741] {standard_task_runner.py:80} INFO - Job 113: Subtask envio_mail
[2023-12-27 18:12:09,967] {task_command.py:371} INFO - Running <TaskInstance: OpenWeather_ETL2.envio_mail manual__2023-12-27T18:11:45.832652+00:00 [running]> on host 44a36bef83ad
[2023-12-27 18:12:10,205] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Furtado_Julio
AIRFLOW_CTX_DAG_ID=OpenWeather_ETL2
AIRFLOW_CTX_TASK_ID=envio_mail
AIRFLOW_CTX_EXECUTION_DATE=2023-12-27T18:11:45.832652+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-12-27T18:11:45.832652+00:00
[2023-12-27 18:12:11,429] {logging_mixin.py:115} INFO - local variable 'add' referenced before assignment
[2023-12-27 18:12:11,432] {logging_mixin.py:115} INFO - Failure
[2023-12-27 18:12:11,437] {python.py:173} INFO - Done. Returned value was: None
[2023-12-27 18:12:11,486] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=OpenWeather_ETL2, task_id=envio_mail, execution_date=20231227T181145, start_date=20231227T181209, end_date=20231227T181211
[2023-12-27 18:12:11,559] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-27 18:12:11,657] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
